src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import "./Footer.scss";
+import { STATIC_DATA } from "../../config/StaticData";
+  
+
+export default function Footer() {
+    const {
+        ENGLISH: {
+            Footer: { FOOTER_CONTACT_INFO_HEADING, FOOTER_CONTACT_INFO, FOOTER_CATEGORY_HEADING, FOOTER_SUBSCRIPTION_HEADING, FOOTER_SUBSCRIPTION_INFO },      
+        },
+      } = STATIC_DATA;
+
+    return(
+        <div>
+        <div id="footer-container">
+            <div className="left">
+                <h2>{FOOTER_CONTACT_INFO_HEADING}</h2>
+                <p>
+                    {FOOTER_CONTACT_INFO}
+                </p>
+            </div>
+            <div className="middle">
+                <h2>{FOOTER_CATEGORY_HEADING}</h2>
+                <ul>
+                    <li>Accesories (25)</li>
```

Comment:
This should be categories that we are showing in header and it should be picked from static data.